packages/formatter-html/src/views/partials/scan-result-item.ejs

```diff
@@ -0,0 +1,49 @@
+<div class="rule-result--details" aria-expanded="false" id="<%= hint.name %>">
+    <div class="rule-result--details__header">
+        <p class="rule-title"><%= hint.name %>: <%= hint.count %> <%= hint.status %><% if(hint.count > 1) { %>s<% } %></p>
+        <div class="rule-result__docs">
+            <a class="docs-button" target="_blank" href="https://webhint.io/docs/user-guide/rules/rule-<%= hint.name %>/" title="documentation for <%= hint.name %> rule">
```

Comment:
Add `rel="noopener noreferrer"`.  `webhint.io` will be a different domain in the case of the formatter output.

Reply:
what do you mean with the different domain? I'm pointing to the website on purpose, do you want to point to the documentation in the hard drive?